models/servicenow__activity_summary.sql

```diff
@@ -4,11 +4,12 @@ with ticket_enhanced as (
     from {{ ref('servicenow__ticket_enhanced') }}
 )
 
-select 
-    task_state,
-    priority,
-    impact,
-    urgency,
+select  
+    cast(task_updated_at as date) as ticket_updated_date,
+    task_state, 
+    priority, 
+    impact, 
+    urgency, 
```

Comment:
Yeah, so I had that question too but ended up deciding not to try mapping it because from what I gathered, the mapping can be customized. For reference, this is how I concluded they're customizable:

https://www.servicenow.com/community/itsm-forum/change-values-for-inc-urgency-and-impact/m-p/865013

UC Berkeley has their own version: https://berkeley.service-now.com/kb_view.do?sysparm_article=KB0010891

Also, Priority is determined by impact/urgency [automatically](https://docs.servicenow.com/bundle/vancouver-it-service-management/page/product/incident-management/task/def-prio-lookup-rules.html) 

But I plan on asking some follow up questions to the customer we initially interviewed; I will include this so they can confirm or direct me to where the mapping is done.